### ISSUE TITLE
Total number of videos N for the load generator

### ISSUE THREAD

**Issue by andreaaraldo** [mora-resource-allocation-edge-cloud/mora#31]
Suppose the load generator wants to generate requests passing through the edge. 
Based on [this page](https://github.com/mora-resource-allocation-edge-cloud/mora/wiki/3.Metrics-collector-and-load-generator#load-generator), we would set `ServiceUrl=<domain_name_of_edge_node>`.

The first thing that the load generator has to know is the number N of total videos in the system. Knowing this information, it can generate videos with ids going from 1 to N (or from 0 to N-1? Please specify this into the doc).

We saw that in the load generator, to know N, a call to `<domain_name_of_edge_node>/videos` is sent, which returns the videos in the edge cache. This is erroneous, as we should instead take all the videos, in the cache and also outside the cache.

We need to add to the load generator a parameter specifying the address of the cloud node

**Comment by andreaaraldo** [comment]
@zazim13  solved it: when the load generator begins, it connects to the cloud and get all the video metadata, so that it can now how many videos are in the catalog.

@zazim13 , please after you commit, add here the link to the line within the load generator that requests such metadata and close the issue

**Comment by zazim13** [comment]
Hi @andreaaraldo  this solution is not finally needed. When the loadgenerator requests for the list of available videos on the system using `<domain_name_of_edge_node>/videos`, so  by requesting the edge node, there is a redirection to the cloud cluster, that is answering with the right list of videos. (For some reason, the cloud wasn't reachable when I did my tests for the first time).

However we spoke about the readability of the loadgenerator code. The solution we already discussed about was to also add the cloud url as a parameter of the loadgenerator. 
`
CloudUrl             = getEnvString("CLOUD_URL", "http://cloud.mora") //The cloud entrypoint `
`
ServiceUrl          = getEnvString("SERVICE_URL", "http://edge.mora") // the loadgenerator will address requests to this entrypoint
`
Is it more readable  or needed ?
